lib/licenses/operators.nix

```diff
@@ -0,0 +1,85 @@
+{
+  /**
+     Create a compound licenses where the user can chose which licenses to use,
+     eqivialent of spdx `or` modifier.
```

Comment:
This should also explain what that operator means for the uninitiated. Ideally just use the upstream definitions and/or link them.

https://spdx.github.io/spdx-spec/v3.0.1/annexes/spdx-license-expressions/